etl/scripts-ccao-data-warehouse-us-east-1/sale/sale-flag_override.R

```diff
@@ -0,0 +1,114 @@
+library(readxl)
+library(janitor)
+library(dplyr)
+library(arrow)
+library(aws.s3)
+source("utils.R")
+
+s3_dir <- "s3://ccao-data-raw-us-east-1/sale/flag_override/"
+
+objs <- get_bucket(
+  bucket = "ccao-data-raw-us-east-1",
+  prefix = "sale/flag_override/"
+)
+
+tmp_dir <- tempdir()
+
+dfs <- list()
+
+for (obj in objs) {
+  key <- obj[["Key"]]
+  if (is.null(key) || grepl("/$", key)) next # skip directory placeholders
```

Comment:
I believe this line is needed because `get_bucket` also returns prefix objects that end in `/`